lean/main/intro.lean

```diff
@@ -131,15 +131,15 @@ elab "#assertType " termStx:term " : " typeStx:term : command =>
 #assertType 5  : Nat -- success
 #assertType [] : Nat -- failure
 
-/-! We started by using `elab` to define a `command` syntax, which, when parsed
+/-! We started by using `elab` to define a `command` syntax, which when parsed
 by the compiler, will trigger the incoming computation.
 
 At this point, the code should be running in the `CommandElabM` monad. We then
 use `liftTermElabM` to access the `TermElabM` monad, which allows us to use
-`elabType` and `elabTermEnsuringType` in order to build expressions out of the
+`elabType` and `elabTermEnsuringType` to build expressions out of the
 syntax nodes `typeStx` and `termStx`.
 
-First we elaborate the expected type `tp : Expr` and then we use it to elaborate
+First, we elaborate the expected type `tp : Expr` and then we use it to elaborate
```

Comment:
I don't know that I'm a fan of this comma addition, to me it reads better without it, or alternatively as

```suggestion
First, we elaborate the expected type `tp : Expr`, then we use it to elaborate
```
The sentence is slightly awkwardly long too I guess.

Reply:
Rewritten the sentence a bit. Please let me know if you like this better.